src/MoBi.UI/Views/EditParametersInContainerView.cs

```diff
@@ -193,15 +209,38 @@ public override void InitializeBinding()
             .WithRepository(dto => _removeButtonRepository)
             .WithFixedWidth(OSPSuite.UI.UIConstants.Size.EMBEDDED_BUTTON_WIDTH);
 
+         gridView.ShowingEditor += (o, e) => OnEvent(onShowingEditor, e);
+         gridView.RowStyle += (o, e) => OnEvent(onRowStyle, e);
+
          _removeButtonRepository.ButtonClick += (o, e) => OnEvent(_presenter.RemoveParameter, _gridViewBinder.FocusedElement);
 
 
          gridView.FocusedRowChanged += (o, e) => OnEvent(gridViewRowChanged, e);
 
          chkShowAdvancedParameter.CheckedChanged += (o, e) => OnEvent(showAdvancedParameterChanged);
          chkGroupBy.CheckStateChanged += (o, e) => OnEvent(groupByChanged);
-         btAddParameter.Click += (o, e) => OnEvent(_presenter.AddParameter);
-         btLoadParameter.Click += (o, e) => OnEvent(_presenter.LoadParameter);
+         btnAddParameter.Click += (o, e) => OnEvent(_presenter.AddParameter);
+         btnLoadParameter.Click += (o, e) => OnEvent(_presenter.LoadParameter);
+
+         _screenBinder.Bind(x => x.SelectedIndividual).To(cbSelectIndividual).WithValues(x => x.AllIndividuals).OnValueUpdated += (o, e) => OnEvent(_presenter.UpdatePreview);
```

Comment:
Can we not put in in one line so that it respects the typical one instruction per line?